modules/kms-keyring-browser/test/kms_keyring_browser.test.ts

```diff
@@ -5,7 +5,9 @@
 
 import * as chai from 'chai'
 import chaiAsPromised from 'chai-as-promised'
-import { KmsKeyringBrowser, getClient, KMS } from '../src/index'
+import { KmsKeyringBrowser, getClient } from '../src/index'
+import { KMS as V2KMS } from 'aws-sdk'
+import { KMS as V3KMS } from '@aws-sdk/client-kms'
```

Comment:
Question: 
If we do not need `import { KMS } from 'aws-sdk'` in any of the source files,
[should we remove `"aws-sdk"` from `kms-keyring-browser`'s `package.json`](https://github.com/aws/aws-encryption-sdk-javascript/blob/fd8264a3c4a0a34f8e554d56caedbdde2d7bf7d8/modules/kms-keyring-browser/package.json#L25)?

Or can we declare it a test-only dependency?

Reply:
Unfortunately we need to keep it for now.
This is because a customer may be getting their `aws-sdk` dependency from this package.

We can remove it on the next major version.